src/scss/06_components/_tags.scss

```diff
@@ -0,0 +1,53 @@
+$tag-gap: $spacing-half;
+
+.content-tags__tags {
+  @include flex-gap($tag-gap);
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  margin-bottom: 0;
+  margin-top: 0;
+  padding-bottom: ($spacing-and-half - $tag-gap);
+  padding-left: 0;
+  padding-top: $spacing-and-half;
+}
+
+.content-tags__tags__tag {
+  list-style: none;
+  margin-bottom: $tag-gap;
+  margin-top: 0;
+
+  a,
+  span {
+    @include font('small');
+    display: inline-block;
+    line-height: 12px;
```

Comment:
This is a major no-no on multiple fronts:
* We should never have line-height that's smaller than font-size, it can cause all sorts of problems. Starting with text being too close to the line above it and ascenders/descenders colliding.
![image](https://user-images.githubusercontent.com/1191667/175276863-b472eb71-b167-44cd-8254-533ede9ad6b4.png) 
![image](https://user-images.githubusercontent.com/1191667/175277859-a9606287-ccf6-4e9a-a6a6-6815172cf684.png)

* We should not define font tokens like this. Instead we should be using the ready made font tokens, like you're using with `font('small')` above.
* We clearly have the space for the height, as you're setting padding for this element below. Instead remove this `line-height: 12px` and adjust padding accordingly (to 2px top and bottom)
* We should not set regular line-height in px ever. We should always go towards relative height compared to em size, like 1.5 should be our minimum for most cases for small font-sizes (accessibility requirement).
* Here is the same example above with line-height removed and top and bottom padding set to 2px
![image](https://user-images.githubusercontent.com/1191667/175278057-d4b59913-5509-49c0-91d9-8077b6756f85.png)
![image](https://user-images.githubusercontent.com/1191667/175278100-33ccd193-ad52-455a-a809-fe0a7e85d9c2.png)